downloads/id/github-git-cheat-sheet.md

```diff
@@ -50,43 +53,44 @@ Unduh sebuah proyek dan seluruh riwayat revisinya
 
 {% capture colTwo %}
 
-## Buat perubahan
-Tinjau suntingan dan daftarkan _commit_
+## Membuat Perubahan
+Tinjau suntingan dan buat sebuah transaksi _commit_
 
 ```$ git status```
 
-Daftar semua berkas yang baru dibuat atau telah dimodifikasi dan akan didaftarkan dalam bentuk _commit_
+Daftar semua berkas baru atau modifikasi yang siap didaftarkan dalam _commit_
```

Comment:
Menampilkan daftar semua berkas baru atau modifikasi yang siap didaftarkan dalam _commit_ much proper because Lists is a verb